app/experimental_appconfig.go

```diff
@@ -47,3 +49,9 @@ func setCustomModules(app *RegenApp, interfaceRegistry types.InterfaceRegistry)
 	}
 	/* New Module Wiring END */
 }
+
+func (app *RegenApp) registerUpgradeHandlers() {
+	app.UpgradeKeeper.SetUpgradeHandler("Gir", func(ctx sdk.Context, plan upgradetypes.Plan) {
+		// no-op handler, does nothing
```

Comment:
would be good to add a comment on why we are adding a no-op upgrade handler, for those who have no context (like me).